notebooks/hyperkitty_to_csv.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>this threw an error for me since you already removed(name) above in an earlier cell. Do you want to delete this cell? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>